apps/api/src/user/service/user.service.ts

```diff
@@ -32,6 +41,28 @@ export class UserService {
       profilePictureUrl: dto?.profilePictureUrl,
       isOnboardingFinished: dto.isOnboardingFinished
     }
+    if (dto?.email) {
+      const userwithEmail = await this.prisma.user.findFirst({
```

Comment:
Renamve this variable to `userExists` and make it boolean using `count` or `findUnique`